operations/sales/partner-programs.md

```diff
@@ -114,10 +114,9 @@ For example: A well-known city with a unique name like Johannesburg would be dis
 
 ## Partner Evaluation License
 
-As a Value-Added Reseller or Deployment Solution Partner, you can request a renewable 1-year E20 license for up to 50 users.
+As a Value-Added Reseller or Deployment Solution Partner, you can request a renewable 1-year [Mattermost Enterprise](https://mattermost.com/pricing) license for up to 50 users.
 
-* Go to [https://www.mattermost.com/trial/](https://www.mattermost.com/trial/) to complete a trial form and enter \[RESELLER\] after your organization name.
-* In the “Where are you with Mattermost” section, select **Other**.
+* Go to [https://www.mattermost.com/trial/](https://www.mattermost.com/trial/) to complete a trial form and enter \[RESELLER\] after your Company Name.
 * Please send an email to partners@mattermost.com letting us know the above have been completed so your application can be reviewed and matched against our certified reseller database.
```

Comment:
@jasonblais @hannaparks Why submit the form if they still have to email us? Can we drop a step?

Reply:
It would save us asking them for their info?  But I'm 0/5, I'd like to make a separate trial/evaluation page for partners in the near future, what do you think @azigler

Reply:
I'm still confused -- what information do we have to ask them for that we can't just collect in the form?

Reply:
Right now, this goes to trials which are a 14 or 30 day trial license.  Only if they email us will we know that they are a partner and the easiest way for them right now is to send us an email after submitting the form.  Ideally we should have a separate page for them but as of now we don't :(

Reply:
@hannaparks brainstorming - what if we created our own form? Or alternatively, instead of having them to first submit a form then email us, just having them email us the info?

0/5, brainstorming to see if there are ways that simplify the process for the partner

Reply:
Oh like on Zendesk? Like the NP form?  I like it, good idea!

Reply:
+1 to Jason's idea, anything to simplify the process for the partner here is best.

Reply:
I have a call with Emily this Friday to discuss the current workflows,  I'll see if we can also add the Zendesk form but it may have to be a completely separate workflow or change the current workflow.
NS: HP to create first draft of ZD form to share with Emily on Mar.8.

Reply:
Moved to https://github.com/mattermost/mattermost-handbook/pull/1336